ts/features/payments/bizEventsTransaction/components/PaymentsBizEventsTransactionHeadingSection.tsx

```diff
@@ -81,24 +91,27 @@ export const PaymentsBizEventsTransactionHeadingSection = ({
   const totalAmount = calculateTotalAmount(transactionInfo);
 
   return (
-    <View style={[IOStyles.horizontalContentPadding, IOStyles.bgWhite]}>
+    <View style={[IOStyles.horizontalContentPadding, { backgroundColor }]}>
       <PaymentsBizEventsTransactionCartList
         carts={transaction?.carts}
         loading={isLoading}
         onPress={handlePressTransactionDetails}
       />
       <VSpacer size={8} />
-      {totalAmount && (
-        <>
+      <VStack space={8}>
+        <PaymentsBizEventsTransactionCartList
+          carts={transaction?.carts}
+          loading={isLoading}
+          onPress={handlePressTransactionDetails}
+        />
+        {totalAmount && (
           <PaymentsBizEventsTransactionTotalAmount
             loading={isLoading}
             totalAmount={totalAmount}
           />
-          <VSpacer size={8} />
-        </>
-      )}
-      <FeeAmountSection />
-      <VSpacer size={8} />
+        )}
+        <FeeAmountSection />
+      </VStack>
     </View>
```

Comment:
Watch out, there is a duplication of the `PaymentsBizEventsTransactionCartList` component here and if you open a transaction details from the app, it will show two items in the list

Reply:
@Hantex9 Totally missed this duplication, thanks 🙏🏻  Fixed in the following commit: [`93714ba` (#6285)](https://github.com/pagopa/io-app/pull/6285/commits/93714bac776bf2a4788028a11da3c6e56c754a0b).

Browsing through the app, I've also replaced the `GradientScrollView` in the `WalletPaymentDetailScreen` because it wasn't dark mode compliant → https://github.com/pagopa/io-app/commit/0e18f33b16f47f5822ab52bee953268536d2e3a0
Noticed that there was a missing translation, _**Vai al pagamento**_: if you have a common i18n string it would be great to fix this.

Reply:
Thank you! I'll address it in another PR 🚀